src/views/share/page.ejs

```diff
@@ -18,6 +18,8 @@
 <body>
     <div id="layout">
         <div id="main">
+            <br>
+            <a href="<%= note.parents[0].noteId %>">&lt; Parent note (<%= note.parents[0].title %>)</a>
```

Comment:
Include parent note button ![](https://user-images.githubusercontent.com/69441971/147395848-c4710b15-708a-432a-b510-4b163f970a54.png)

Reply:
* check that there is a parent (root note doesn't have one)
* I would put this into `<nav class="parent-link">` for semantics and styling

Reply:
I checked if the parent note's ID == "share" and if the parent array is 0.